packages/core/src/components/slider/multiSlider.tsx

```diff
@@ -0,0 +1,272 @@
+        }
+
+        return <div className={Classes.SLIDER_TRACK}>{tracks}</div>;
+    }
+
+    protected renderHandles() {
+        const { disabled, max, min, stepSize, vertical } = this.props;
+        return this.sortedHandleProps.map(({ value, type }, index) => (
+            <Handle
+                className={classNames({
+                    [Classes.START]: type === SliderHandleType.START,
+                    [Classes.END]: type === SliderHandleType.END,
+                })}
+                disabled={disabled}
+                key={`${index}-${this.sortedHandleProps.length}`}
+                label={this.formatLabel(value)}
+                max={max}
+                min={min}
+                onChange={this.getHandlerForIndex(index, this.handleChange)}
+                onRelease={this.getHandlerForIndex(index, this.props.onRelease)}
+                ref={this.addHandleRef}
+                stepSize={stepSize}
+                tickSize={this.state.tickSize}
+                tickSizeRatio={this.state.tickSizeRatio}
+                value={value}
+                vertical={vertical}
+            />
+        ));
+    }
+
+    protected handleTrackClick(event: React.MouseEvent<HTMLElement>) {
+        const foundHandle = this.nearestHandleForValue(this.handles, handle => handle.mouseEventClientOffset(event));
+        if (foundHandle) {
+            foundHandle.beginHandleMovement(event);
+        }
+    }
+
+    protected handleTrackTouch(event: React.TouchEvent<HTMLElement>) {
+        const foundHandle = this.nearestHandleForValue(this.handles, handle => handle.touchEventClientOffset(event));
+        if (foundHandle) {
+            foundHandle.beginHandleTouchMovement(event);
+        }
+    }
+
+    private renderTrackFill(index: number, start: ISliderHandleProps, end: ISliderHandleProps, intent: Intent) {
+        const { tickSizeRatio } = this.state;
+        const startValue = start.value;
+        const endValue = end.value;
+
+        let startOffsetRatio = this.getOffsetRatio(startValue, tickSizeRatio);
+        let endOffsetRatio = this.getOffsetRatio(endValue, tickSizeRatio);
+
+        if (startOffsetRatio > endOffsetRatio) {
+            const temp = endOffsetRatio;
+            endOffsetRatio = startOffsetRatio;
+            startOffsetRatio = temp;
+        }
+
+        const startOffset = formatPercentage(startOffsetRatio);
+        const endOffset = formatPercentage(1 - endOffsetRatio);
+
+        const style: React.CSSProperties = this.props.vertical
+            ? { bottom: startOffset, top: endOffset, left: 0 }
+            : { left: startOffset, right: endOffset, top: 0 };
+
+        const classes = classNames(Classes.SLIDER_PROGRESS, intentClass(intent), {
+            [Classes.START]: start.type === SliderHandleType.START,
+            [Classes.END]: end.type === SliderHandleType.END,
+            [`${Classes.SLIDER_PROGRESS}-empty`]: intent === Intent.NONE,
```

Comment:
this class is not essential. intent can override background color

Reply:
This was previously required, need to rethink if its still needed with the removal of other slider implementations.